design/vela-core/environment.md

```diff
@@ -0,0 +1,213 @@
+# Managing Environments As Shared Bases For Application Deployment
+
+## Background
+
+A team of application development usually needs to prepare some shared environments for developers to deploy applications to.
+For example, most teams will have two environments, a *dev* environment for testing applications,
```

Comment:
```suggestion
For example, most teams will have two environments, a *staging* environment for testing applications,
```
Normally we call it testing or staging environment.

Reply:
I would like to keep it as dev which is simpler.